pyaqsapi/textvariables.py

```diff
@@ -0,0 +1,115 @@
+"""textvariables.py"""
+
+# Text strings to allow simpler input into `pyaqsapi.bystate()` 
+Alabama =  '01'    
+AL =  '01'
+Alaska =  '02'    
+AK =  '02'
+Arizona =  '04'    
+AZ =  '04'
+Arkansas =  '05'    
+AR =  '05'
+California =  '06'    
+CA =  '06'
+Colorado =  '08'    
+CO =  '08'
+Connecticut =  '09'   
+CT =  '09'
+Delaware =  '10'   
+DE =  '10'
+DistrictOfColumbia =  '11'   
+DC =  '11'
+Florida =  '12'   
+FL =  '12'
+Georgia =  '13'   
+GA =  '13'
+Hawaii =  '15'   
+HI =  '15'
+Idaho =  '16'    
+ID =  '16'
+Illinois =  '17'  
+IL =  '17'
+Indiana =  '18'   
+IN =  '18'
+Iowa =  '19'  
+IA =  '19'
+Kansas =  '20'   
+KS =  '20'
+Kentucky =  '21'  
+KY =  '21'
+Louisiana =  '22'   
+LA =  '22'
+Maine =  '23'   
+ME =  '23'
+Maryland =  '24'   
+MD =  '24'
+Massachusetts =  '25'   
+MA =  '25'
+Michigan =  '26'   
+MI =  '26'
+Minnesota =  '27'   
+MN =  '27'
+Mississippi =  '28'  
+MS =  '28'
+Missouri =  '29'   
+MO =  '29'
+Montana =  '30'    
+MT =  '30'
+Nebraska =  '31'   
+NE =  '31'
+Nevada =  '32'  
+NV =  '32'
+NewHampshire =  '33' 
+NH =  '33'
+NewJersey =  '34'  
+NJ =  '34'
+NewMexico =  '35'   
+NM =  '35'
+NewYork =  '36'   
+NY =  '36'
+NorthCarolina =  '37' 
+NC =  '37'
+NorthDakota =  '38'   
+ND =  '38'
+Ohio =  '39'    
+OH =  '39'
+Oklahoma =  '40'   
+OK =  '40'
+Oregon =  '41'
+OR =  '41'
+Pennsylvania =  '42'   
+PA =  '42'
+RhodeIsland =  '44'   
+RI =  '44'
+SouthCarolina =  '45'   
+SC =  '45'
+SouthDakota =  '46'   
+SD =  '46'
+Tennessee =  '47'  
+TN =  '47'
+Texas =  '48' 
+TX =  '48'
+Utah =  '49' 
+UT =  '49'
+Vermont =  '50' 
+VT =  '50'
+Virginia =  '51'  
+VA =  '51'
+Washington =  '53' 
+WA =  '53'
+WestVirginia =  '54'  
+WV =  '54'
+Wisconsin =  '55'   
+WI =  '55'
+Wyoming =  '56' 
+WY =  '56'
+Guam =  '66' 
+GU =  '66'
+PuertoRico =  '72'
+PR =  '72'
+VirginIslands =  '78'  
+VI =  '78'
+Mexico =  '80'
+MX =  '80'
+Canada =  'CC'   
+CAN =  'CC'
```

Comment:
why "CC" instead of another number? Mexico right above for example uses 80.

you might also think about using python enums:

https://docs.python.org/3/library/enum.html

(There is even an auto() now to avoid having to number things.. though if PR becomes a state in the future and is added maybe having things auto numbered may be an issue? Not sure)

Another thought is using some sort of library for ISO country and state names like:

https://pypi.org/project/geonamescache/

Though this may be overkill

Reply:
These values are actually what the API expects when inputting the `stateFIPS` variable.

They are available when calling `aqs.aqs_states()` to search for what number you would need when using the `aqs.bystate()` function to request data for a specific state. I just proposed this addition so they can be input on the user end by their name or abbreviation and still have the API receive the code it expects.

Reply:
Ah okay, I see. That makes sense!